### PR TITLE
Improved layout for data owned by alice

### PR DESCRIPTION
**Pull request by elf-pavlik**
[**rendered preview**](https://deploy-preview-119--data-interoperability-panel.netlify.app/primer/)

This PR includes minimum cleanup and marks many snippets as "needs to be updated".

It focuses on new layout for Alice's data, similar to one mentioned in #113 

I avoid getting into remote data registration. Next step will be few different data grants where Alice gives different combination of access for data she owns to specific application she wants to use.

I'm also introducing bridge/join resources where each *parent* resource can keep references to it's *children*.

### CODE REVIEW

File: proposals/primer/snippets/prefixes.ttl [hunk 1]
```diff
@@ -4,9 +4,23 @@
 @prefix dcterms: <http://purl.org/dc/terms/> .
 @prefix solid: <http://www.w3.org/ns/solid/terms#> .
 @prefix interop: <http://www.w3.org/ns/solid/interop#> .
+
```
Comment (justinwb) [inline proposals/primer/snippets/prefixes.ttl:7]: Should also include the prefix for shape trees. We typically are using:

`@prefix st: <http://www.w3.org/ns/st#>`

File: proposals/primer/snippets/solidshapes.example/shapes/project.shex [hunk 2]
```diff
@@ -1,6 +1,24 @@
...
+  a [ pm:Project ] ;
+  rdfs:label xsd:string ;
+  tbd:hasReferencesList @<#TaskReferencesList> ;
+  tbd:hasReferencesList @<#MilestoneReferencesList>
+}
+
+<#TaskReferencesList> {
+  a [ tbd:ReferencesList ] ;
+  tbd:referencesOf @solidshapes:ProjectShape ;
+  tbd:referenceShapePath ["@solidshapes:Project/pm:hasTask/@solidshapes:Task"]
```
[the remark this answers is not in the document]
Comment (elf-pavlik) [reply proposals/primer/snippets/solidshapes.example/shapes/project.shex:17]: Shape tree definition can have any number of values for `st:reference`, I think each `tbd:ReferencesList` needs to describe which one of those relationship types it contains - `pm:hasTask`, `pm:hasMilesone` etc.

File: proposals/primer/snippets/solidshapes.example/trees/project.ttl [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
 solidtrees:Project
   a shapetree:ShapeTree ;
   shapetree:expectsType shapetree:ShapeTreeResource ;
   shapetree:validatedBy solidshapes:Project ;
-    shapetree:reference [
-    shapetree:traverseViaReverseShapePath "@solidshapes:Task/ex:inProject" ;
-    shapetree:hasShapeTree solidtrees:Task
-  ] ;
-  shapetree:matchesUriTemplate "{uuid}" .
+  shapetree:matchesUriTemplate "{uuid}" ;
```
[the remark this answers is not in the document]
Comment (elf-pavlik) [reply proposals/primer/snippets/solidshapes.example/trees/project.ttl]: removed

File: proposals/primer/snippets/prefixes.ttl (hunk 1 above)
Comment (elf-pavlik) [reply proposals/primer/snippets/prefixes.ttl:8]: added `@prefix shapetree: <http://www.w3.org/ns/shapetree#> .` as used in all the snippets.

**Review by justinwb — APPROVED** [review]
Approval following discussion in 6/1 panel

**Pull request merged**